pkg/systemtemplate/template.go

```diff
@@ -315,7 +320,7 @@ spec:
   updateStrategy:
     type: RollingUpdate
     rollingUpdate:
-      maxUnavailable: 25%
+      maxUnavailable: 50%
```

Comment:
This is to speed up deployment I assume?

Reply:
Yes, this is just to accelerate the rollout